cilium/cmd/debuginfo.go

```diff
@@ -265,3 +443,26 @@ func writeMarkdown(data []byte, path string) {
 	w := tabwriter.NewWriter(f, 5, 0, 3, ' ', 0)
 	w.Write(data)
 }
+
+func writeJSON(data []byte, path string) {
+	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0600)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Could not create file %s", path)
+		os.Exit(1)
+	}
+
+	db := &models.DebugInfo{}
+
+	err = db.UnmarshalBinary(data)
```

Comment:
Is the unmarshal/marshal just so that we can get some indentation?

Reply:
Yes.